hugegraph-api/src/main/java/com/baidu/hugegraph/serializer/JsonSerializer.java

```diff
@@ -76,7 +77,8 @@ private String writeList(String label, List<?> list) {
         }
     }
 
-    private String writeList(String label, Iterator<?> itor, boolean paging) {
+    private String writeIterator(String label, Iterator<?> itor,
```

Comment:
also update writeList(String label, List<?> list) to call JsonUtil.toJson(), keep writeList()